package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@apify/eslint-config",
-  "version": "0.3.4",
+  "version": "0.3.5",
```

Comment:
maybe this change deserves a minor bump? it will produce quite a lot of errors in most of the codebases

Reply:
yeah, true. went for `0.4.0`.